tmpl/file.go

```diff
@@ -15,6 +15,7 @@ type templateFileRenderer struct {
 
 type TemplateData struct {
 	// Environment is accessible as `.Environment` from any template executed by the renderer
+	// Namespace is accessible as `.Namespace` from any non-values template executed by the renderer
```

Comment:
For go doc to operate correctly, this has to be on top of the documented field.

Reply:
Huh okay. Fixed!